apps/src/music/views/MusicView.jsx

```diff
@@ -884,6 +887,21 @@ class UnconnectedMusicView extends React.Component {
     });
     if (this.props.isFirstAttempt) {
       this.props.sendAttemptReport();
+
+      const eventName = this.props.levelProperties.isProjectLevel
+        ? EVENTS.PROJECT_ACTIVITY
+        : EVENTS.LEVEL_ACTIVITY;
+
+      analyticsReporter.sendEvent(
```

Comment:
Flagging for @dju90 - note that this metric will forward to the non-Music Lab, generic code.org Amplitude project (in case it's relevant before we move over to statsig).